intercom_flutter_web/test/tests_exist_elsewhere_test.dart

```diff
@@ -0,0 +1,14 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
```

Comment:
I think this `test/` folder can be removed

Reply:
no need to remove it. It will tell the user that this package uses integration_tests